cmd/seed.go

```diff
@@ -0,0 +1,196 @@
+package cmd
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	shieldv1beta1 "github.com/raystack/shield/proto/v1beta1"
+	cli "github.com/spf13/cobra"
+	"google.golang.org/grpc/metadata"
+)
+
+var (
+	//go:embed seed/permissions.json
+	mockCustomPermissions []byte
+	//go:embed seed/roles.json
+	mockCustomRoles []byte
+	//go:embed seed/user.json
+	mockHumanUser []byte
+	//go:embed seed/organization.json
+	mockOrganization []byte
+	//go:embed seed/project.json
+	mockProject []byte
+	//go:embed seed/resource.json
+	mockResource []byte
+
+	orgAdminEmail  = "sampleAdmin@raystack.org"
```

Comment:
`orgAdminEmail` looks very general to shield, we should name it specific for seed command. For example: `sampleSeedEmail`? Also camel casing email doesn't look good, how about `sample.admin@raystack.org`